xep-0404.xml

```diff
@@ -132,29 +144,29 @@ This change means that the client will not be able to determine real JID of the
 
      <p>
        When JIDs are being hidden, the resource of the full JIDs stored in the presence node MUST also be anonymized using a similar mechanism.
-      First the bare JID in presence is a proxy JID, as defined in &xep0369;.   Where the JID is not being hidden, the resource is simply the resource of the clients full JID.   Where the JID is hidden,   the resource is replaced with a generated value.   For example,   'hag66@shakespeare.example/UUID-a1j/7533' in the channel coven might have a proxy JID of '123456#coven@mix.shakespeare.example/789'.   There is no client visible mapping of proxy full JIDs maintained as this is not needed.   The MIX channel will need to maintain a mapping, to support directly addressing clients, such as for client to client disco#info queries.  While an full proxy JID is held in the presence node, the mapping to real JID MUST NOT be changed.  When adding a client to the presence node, the server MAY add the same anonymized JID as used before by that client, or it MAY create a different anonymized JID.
+   Where the JID is not being hidden, the resource is simply the resource of the clients full JID.   Where the JID is hidden,   the resource is replaced with a generated value.   For example,   'hag66@shakespeare.example/UUID-a1j/7533' in the channel coven might have an encoded  JID of '123456#coven@mix.shakespeare.example/789'.   There is no client visible mapping maintained, as this is not needed.   The MIX channel will need to maintain a mapping, to support directly addressing clients, such as for client to client disco#info queries.  While an encoded JID is held in the presence node, the mapping to real JID MUST NOT be changed.  When adding a client to the presence node, the server MAY add the same anonymized JID as used before by that client, or it MAY create a different anonymized JID.
```

Comment:
client-id should be used even when it's not JID-hidden (see other PR's comments).